tests/test_transcription.py

```diff
@@ -0,0 +1,27 @@
+# CREATED: 2/9/16 2:27 PM by Justin Salamon <justin.salamon@nyu.edu>
+
+import mir_eval
+
+
+def test_precision_recall_f1():
+
+    ref_int, ref_pitch = mir_eval.io.load_valued_intervals(
+        'tests/data/transcription/ref00.txt')
+    est_int, est_pitch = mir_eval.io.load_valued_intervals(
+        'tests/data/transcription/est00.txt')
+
+    precision, recall, f_measure = \
+        mir_eval.transcription.precision_recall_f1(ref_int, ref_pitch, est_int,
+                                                   est_pitch)
+
+    assert precision == 3/5.
+    assert recall == 3/4.
+    assert f_measure == 2 * precision * recall / (precision + recall)
```

Comment:
Haha, so, since it turns out this is a unit test, not a regression test, this should actually go in the `test_transcription.py` file itself.  The point is that it should either (as a regression test) all be in reference/estimated text annotation files and expected output JSON files, OR be hardcoded (as a unit test) in the test_*.py file.  You can change this when you have real-world data to test on.